build-system/tasks/dist.js

```diff
@@ -96,6 +97,7 @@ async function runPreDistSteps(options) {
   await compileJison();
   await copyParsers();
   await bootstrapThirdPartyFrames(options);
+  await minifyLocaleJson();
```

Comment:
Nit: It seems we could build this into `build-system/babel-plugins/babel-plugin-transform-json-import/index.js`, which would allow us to support the dev server without any changes. It'd also allow us to actually test, instead of hitting some bug when we push a release.

Reply:
Now built into the babel plugin, and added a test. Thanks for the suggestion!